ast.c

```diff
@@ -274,6 +274,12 @@ static const char *visibility_flags[] = {
 	NULL
 };
 
+static const char *class_const_flags[] = {
+	AST_VISIBILITY_FLAGS,
+	AST_FLAG(MODIFIER_FINAL),
```

Comment:
Technically, as far as parsing is concerned, it's possible for all AST_MODIFIER_FLAGS to occur here. STATIC/ABSTRACT are rejected during compilation. Maybe we want to skip the distinction between modifier_flags and class_const_flags?

Reply:
>  Maybe we want to skip the distinction between modifier_flags and class_const_flags?

Makes sense